### ISSUE TITLE
Suggestion not an issue

### ISSUE THREAD

**Issue by nooga22** [Shaztech/Pilldispenser#19]
Maybe add a screen lock option that requires a pin to unlock for people with children.

**Comment by Shaztech** [comment]
Good idea, I will add that in the next release, But the password will only be configurable in the web interface, so you cannot lock yourself out and always change the code.

**Comment by Shaztech** [comment]
Check version 1.8, it's now integrated. 

![image](https://github.com/user-attachments/assets/89ed489b-cffa-45c1-ba71-f1a6402bbf1f)
![image](https://github.com/user-attachments/assets/6efc92b7-32bb-4425-b81e-06cc2e90bd95)
